mooncake-store/src/master_service.cpp

```diff
@@ -304,6 +295,10 @@ ErrorCode MasterService::PutStart(
         replica_list.emplace_back(replica.get_descriptor());
     }
 
+    // Set lease timeout to now, indicating that the object has no lease
+    // at beginning
+    metadata.lease_timeout = std::chrono::steady_clock::now();
```

Comment:
If we're using the default values from the constructor like we talked above, this code here might be redundant.

Reply:
The current eviction policy selects the objects with the oddest lease time out to evict.  Here we set the lease_timeout to now instead of zero, so if some objects have not been read since beginning, the one put first are evicted first.